client/my-sites/stats/features/modules/stats-utm/stats-module-utm.jsx

```diff
@@ -263,4 +279,7 @@ const StatsModuleUTM = ( {
 	);
 };
 
-export { StatsModuleUTM as default, OPTION_KEYS };
+const MemoizedStatsModuleUTM = memo( StatsModuleUTM );
```

Comment:
Can you please add more information why this could be beneficial here?

Reply:
Thank you for pointing out that this is actually leftover change when I was trying to isolate another error, which was causing errors due to repeat dispatches, and should be removed